src/app/+item-page/simple/item-types/publication/publication.component.html

```diff
@@ -28,19 +28,24 @@ <h2 class="item-page-title-field">
   </div>
   <div class="col-xs-12 col-md-6">
     <ds-metadata-representation-list
-      [label]="'relationships.isAuthorOf' | translate"
-      [representations]="authors$ | async">
+      [parentItem]="item"
```

Comment:
Hi @Atmire-Kristof , this isn't working for me. I've already started a project from scratch, loaded some entities, but still I can't see related authors. Meanwhile on the author's page, I can see the publications. Am I missing something?

Reply:
I double-checked, but can't seem to reproduce your issue. I am able to see a list of author relations on publication pages. I've tested with both the 4science rest api (dspace7.4science.cloud), as well as our own.

Reply:
This is a part of full item page, where you can see the relationship:
![2019-10-15 12_03_47-11-ketotestosterone inhibits the alternative mating tactic in sneaker males of t](https://user-images.githubusercontent.com/877488/66826631-d0b75080-ef44-11e9-8573-d403b1776463.png)

And this is the simple page, with no reference to the author:
![2019-10-15 12_04_27-11-ketotestosterone inhibits the alternative mating tactic in sneaker males of t](https://user-images.githubusercontent.com/877488/66826693-f2b0d300-ef44-11e9-938d-d3de8471383b.png)

Reply:
As far as the two pictures go, I don't see anything wrong here.

There's two reasons I can think of that could cause this issue:
- Your publication doesn't contain a "virtual metadata value" for the author. There should be a metadata value under `dc.contributor.author` with an authority property that links to the `relation.isAuthorOfPublication` metadata (e.g. they both have authority property set to `virtual::1234`).
- Your REST API doesn't have the `relationships/search/byLabel` endpoint deployed (as described [here](https://github.com/DSpace/Rest7Contract/blob/master/relationships.md#relationships-per-relationship-type). However, this would most likely cause a bunch of console or other errors.

If those aren't the case, maybe you could make sure I can access the item for myself to take a better look, as I don't think this item is accessible on the 4science REST API?

Reply:
I'm getting empty Person's pages with 4Science endpoint:
![image](https://user-images.githubusercontent.com/877488/67225696-9578be00-f42b-11e9-9585-c65ab7fb6282.png)
I will start all over again.

Reply:
I've just forced this on the template:
<ds-related-items
      [parentItem]="item"
      [relationType]="'isAuthorOfPublication'"
      [label]="'relationships.isAuthorOf' | translate">
</ds-related-items>

And the related author is showed. I've also this metadata in my response:
```
"metadata" : {
    "dc.contributor.author" : [ {
      "value" : "Oliveira, R. F.",
      "language" : null,
      "authority" : null,
      "confidence" : -1,
      "place" : 0
    }, {
      "value" : "Canario, Adelino",
      "language" : "*",
      "authority" : "virtual::21",
      "confidence" : -1,
      "place" : 1
    }, {
      "value" : "Carneiro, L. A.",
      "language" : null,
      "authority" : null,
      "confidence" : -1,
      "place" : 2
    }, {
      "value" : "Gonçalves, D.",
      "language" : null,
      "authority" : null,
      "confidence" : -1,
      "place" : 3
    }, {
      "value" : "Canario, Adelino",
      "language" : null,
      "authority" : null,
      "confidence" : -1,
      "place" : 4
    }, {
      "value" : "Grober, M. S.",
      "language" : null,
      "authority" : null,
      "confidence" : -1,
      "place" : 5
    } ],
```
So, I've the "authority" : "virtual::21" attribute like you referred. I just think that might be a problem with the refactored: ds-metadata-representation-list component.